nixos/modules/services/misc/lms.nix

```
@@ -0,0 +1,234 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lms;
+in
+{
+  options.services.lms = {
+    enable = mkEnableOption "LMS (Lightweight Music Server). A self-hosted, music streaming server.";
```

```suggestion
    enable = mkEnableOption "LMS (Lightweight Music Server). A self-hosted, music streaming server";
```

Trailing periods are redundant with `mkEnableOption`.

---

Changed.